src/lib/multiply/index.ts

```diff
@@ -13,7 +13,9 @@ export const Multiply = (globalConfig: MultiplyConfig): PluginInterface => {
     kind: 'execute',
   };
 
-  /**Checks global confiog value are valid */
+  /**
+   * Checks global confiog value are valid.
+   */
   const validateGlobalConfig = () => {
     if (inputParameters.length === 0) {
```

Comment:
you can use `zod` as you suggested, like
const configSchema = z.object({
  inputParameters: z.array(z.string()).gt(1),
  outputParameter: z.string().nonempty(),
});